express-post-json/index.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express');
+const app = express();
+
+let nextId = 1;
+const grades = {};
+
+app.get('/api/grades', (req, res) => {
+  const finalGrades = [];
+  for (const key in grades) {
+    finalGrades.push(grades[key]);
+  }
+  res.json(finalGrades);
+});
+
+app.use('/api/grades', express.json());
+
+app.post('/api/grades', (req, res, next) => {
+  const newInfo = req.body;
+  newInfo.id = nextId;
+  grades[nextId] = newInfo;
+  nextId++;
+  res.status(201).send(newInfo);
+
+  next();
```

Comment:
No need to call `next` here, sending the response to the client with `res.status(201).send(newInfo)` is the last thing that needs to happen, so there is no `next`. Also use the `json` method instead of `send`:
```JAVASCRIPT
res.status(201).json(newInfo);
```